apps/functions-subscription/local.settings.json.example

```diff
@@ -6,9 +6,20 @@
     "COSMOSDB_ENDPOINT": "<put-here-the-the-cosmos-database-endpoint>",
     "COSMOSDB_DATABASE_NAME": "<put-here-the-name-of-cosmos-database>",
     "EVENTHUB_NAMESPACE": "<put-here-the-the-event-hub-namespace>",
-    "SUBSCRIPTION_REQUEST_EVENTHUB_NAME": "<put-here-the-name-of-the-event-hub>",
+    "AzureWebJobsStorage": "<put-here-the-connection-string-to-a-storage>",
+
+    "LEASES_COSMOSDB_CONTAINER_NAME": "leases",
+
+    "SUBSCRIPTION_HISTORY_CONSUMER": "on",
+    "SUBSCRIPTION_HISTORY_COSMOSDB_CONTAINER_NAME": "subscription-history",
+    "SubscriptionHistoryCosmosConnection__accountEndpoint": "https://cosmosdb-playground-tmp.documents.azure.com:443/",
```

Comment:
you may want to point to real dev resources on azure so new devs will be able to startup the full environment to develop locally after `az login`

Reply:
I've removed them!